lib/syskit/roby_app/ftp_server.rb

```diff
@@ -0,0 +1,3 @@
+require 'syskit/roby_app/ftp_server/write_only_disk_file_system'
+require 'syskit/roby_app/ftp_server/driver'
+require 'syskit/roby_app/ftp_server/server'
```

Comment:
Naming ... This is not a generic ftp server. Maybe `log_transfer_server` instead ?

Reply:
Generally speaking, names must reflect what something *does*, not what it *is*.